test.py

```diff
@@ -0,0 +1,98 @@
+from boltiot.bolt import Bolt
+import unittest
+import json
+import time
+
+client = Bolt('<your-api-key-here>','<your-bolt-id>') # Pass in the API Key and the client ID.
+
+class BoltTests(unittest.TestCase):
+    
+# Testing the digital write function.
+    
```

Comment:
@vimalsheoran Is there any specific reason you are naming your functions test-1 or test-2. If not then I suggest you name like "test_digital_write".